### PR TITLE
Import the new `Microsoft.Common.Test.targets` target for multi-tfm

### PR DESCRIPTION
**Pull request by MarcoRossignoli**
Should fix https://github.com/dotnet/sdk/issues/37712
image

cc:  @rainersigwald @Forgind   @baronfel  @Evangelink @nohwnd

### CODE REVIEW

File: src/Tasks/Microsoft.Common.CrossTargeting.targets [hunk 1]
```diff
@@ -248,4 +248,9 @@ Copyright (C) Microsoft Corporation. All rights reserved.
 
   <Import Project="$(DirectoryBuildTargetsPath)" Condition="'$(ImportDirectoryBuildTargets)' == 'true' and exists('$(DirectoryBuildTargetsPath)')"/>
 
+  <PropertyGroup>
```
Comment (Forgind) [inline src/Tasks/Microsoft.Common.CrossTargeting.targets:251]: Why do you need this PropertyGroup? Is the property used elsewhere? ('' != 'true', so this shouldn't affect the Import on the next line.)

**Review by Forgind — APPROVED** [review]

File: src/Tasks/Microsoft.Common.CrossTargeting.targets (hunk 1 above)
Comment (MarcoRossignoli) [reply src/Tasks/Microsoft.Common.CrossTargeting.targets:251]: It's the same pattern used here https://github.com/dotnet/msbuild/pull/9193/files#diff-5407d46dd30ce4031e530c35cc2e0a62a6c96e54cb1def14fb316f351ef92de9 the design expect this usage atm
```bash
msbuild /t:Test /p:UseMSBuildTestInfrastructure=true
```

Comment (Forgind) [reply src/Tasks/Microsoft.Common.CrossTargeting.targets:251]: It doesn't look necessary there either, but it isn't really a big deal.

**Comment by MarcoRossignoli** [comment]
some ETA for this one @Forgind ?

**Review by rokonec — APPROVED** [review]

**Pull request merged**